project/AppDependencies.scala

```diff
@@ -5,22 +5,19 @@ object AppDependencies {
 
   private val compile: Seq[ModuleID] = Seq(
     play.sbt.PlayImport.ws,
-    "org.reactivemongo" %% "play2-reactivemongo"            % "0.20.13-play28",
+    "uk.gov.hmrc.mongo" %% "hmrc-mongo-play-28"             % "0.73.0",
     "uk.gov.hmrc"       %% "play-frontend-hmrc"             % "1.1.0-play-28",
-    "uk.gov.hmrc"       %% "play-conditional-form-mapping"  % "1.9.0-play-28",
-    "uk.gov.hmrc"       %% "bootstrap-frontend-play-28"     % "5.24.0",
+    "uk.gov.hmrc"       %% "play-conditional-form-mapping"  % "1.12.0-play-28",
+    "uk.gov.hmrc"       %% "bootstrap-frontend-play-28"     % "7.8.0",
     "uk.gov.hmrc"       %% "tax-year"                       % "1.1.0"
   )
 
   private val test: Seq[ModuleID] = Seq(
     "org.scalatest"               %% "scalatest"                % "3.2.12",
     "org.scalatestplus.play"      %% "scalatestplus-play"       % "5.1.0",
     "org.scalatestplus"           %% "scalatestplus-scalacheck" % "3.1.0.0-RC2",
-    "org.scalatestplus"           %% "mockito-3-12"             % "3.2.10.0",
-    "org.pegdown"                 %  "pegdown"                  % "1.6.0",
-    "org.jsoup"                   %  "jsoup"                    % "1.14.3",
+    "org.mockito"                 %% "mockito-scala-scalatest"  % "1.17.12",
     "com.typesafe.play"           %% "play-test"                % PlayVersion.current,
-    "org.mockito"                 %  "mockito-all"              % "1.10.19",
     "org.scalacheck"              %% "scalacheck"               % "1.16.0",
     "com.github.tomakehurst"      % "wiremock-standalone"       % "2.27.2",
     "com.vladsch.flexmark"        % "flexmark-all"              % "0.62.0"
```

Comment:
"org.scalatest"               %% "scalatest"                % "3.2.12" can be upgraded to 3.2.14
"org.scalacheck"              %% "scalacheck"               % "1.16.0", can be upgraded to 1.16.0